providers/implementations/ciphers/cipher_null.c

```diff
@@ -38,6 +53,14 @@ static OSSL_FUNC_cipher_cipher_fn null_cipher;
 static int null_cipher(void *vctx, unsigned char *out, size_t *outl,
                        size_t outsize, const unsigned char *in, size_t inl)
 {
+    PROV_CIPHER_NULL_CTX *ctx = (PROV_CIPHER_NULL_CTX *)vctx;
+
+    if (!ctx->enc && ctx->tlsmacsize > 0) {
```

Comment:
It probably wouldnt hurt to put in a ref comment.. e.g.

See https://tools.ietf.org/html/rfc5246#section-6.2.3.1

Reply:
Done